packages/frontend/src/components/global/MkMisskeyFlavoredMarkdown.ts

```diff
@@ -102,6 +102,8 @@ export default function(props: MfmProps) {
 			}
 
 			case 'fn': {
+				// https://www.debuggex.com/r/Bb7wXENuNZZzUb2G
+				const rgbaPattern = /^([0-9a-f]{3,4}|[0-9a-f]{6}|[0-9a-f]{8})$/i;
```

Comment:
```suggestion
				const rgbaPattern = /^([\da-f]{3}|([\da-f]{2}){2,4})$/i;
```

per https://github.com/misskey-dev/misskey/issues/10904#issuecomment-1569130438